src/app/auth/login/page.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import * as React from 'react';
+
+import { useSearchParams, useRouter } from 'next/navigation';
+import { signIn } from 'next-auth/react';
+
+import LogInForm from '@/components/Login/Form';
+import toast from '@/lib/toast';
+import { Credentials } from '@/types/common';
+
+const Login = () => {
+  const router = useRouter();
+  const searchParams = useSearchParams();
+
+  const callbackUrl = searchParams.get('callbackUrl') || '/upload';
+
+  const handleFormSubmit = async (payload: Credentials) => {
+    try {
+      await signIn('credentials', {
+        redirect: false,
+        email: payload?.email,
+        password: payload?.password,
+      });
+      toast('Logged in successfully', 'success');
```

Comment:
This message has not been reworked similarly to follow the earlier comments I raised 😓 Ensure messages displayed to the user are well formatted and information.

```suggestion
      toast('You have logged in successfully.', 'success');
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/67/commits/a5f075f3f072945b2440d3f599a2e0ea3948f395